### PR TITLE
About Us page

### PR DESCRIPTION
**Pull request by razo7**
Add an `About Us` page for a better reference of what the team is doing,  who are we, and etc.

It's layout is as follows:
1. Who are we?
2. What we do?
3. What we plan to do?
4. History
5. Community

### CODE REVIEW

File: about_us.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+In 2019, we improved support for bare metal by shipping an
+annotation based mechanism for rebooting nodes instead of going through a time
+expensive reprovisioning cycle.
+
+In 2020, we worked with Ericsson to design an official API for using alternative
+mechanisms to recover bad nodes.  Since then Ericsson has prototyped a
+[metal3](http://metal3.io/) based implementation, and we have implemented
+[Self Node Remediation](https://github.com/medik8s/self-node-remediation) for shared-nothing environments.
+
+In 2021, we created medik8s to make general purpose HA available
```
Comment (razo7) [inline about_us.md]: Have we done more stuff back in 2021?

File: about_us.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+to all kubernetes clusters, not just ones backed by an infrastructure API.
+
```
Comment (razo7) [inline about_us.md]: We need 2022 history

File: about_us.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+We develop Kubernetes Open Source Operators using [Operator Lifecycle Manager (OLM)](https://olm.operatorframework.io/) that provide maintenance support, automatic node remediation and high availability for singleton workloads:
+
+- [Node Healthcheck Operator (NHC)](failure_detection) - Detecting Node Failures, triggering remediation, which is performed by other operators like Self Node Remediation or MachineDeletion.
+- [Self Node Remediation (SNR)](/remediation/self-node-remediation/self-node-remediation) - Remediates nodes by rebooting, without needing a management interface like e.g. IPMI, or a node / machine provisioning API. Works standalone, and / or with NHC.
+- [Node Maintenance Operator (NMO)](maintenance-node) - Declarative node cordoning and draining prior to harmful decisions.
+
+## What we plan to do?
+
+We plan to improve the exiting operators by adding Master (control plane) fencing capabilites to SNR and NHC, which currently **only** work for worker nodes.
```
Comment (razo7) [inline about_us.md:31]: Do we have more tasks that we want to inform publicly?

Comment (beekhof) [reply about_us.md:31]: I think that would be a good topic for our f2f :)

File: about_us.md (hunk 1 above)
Comment (beekhof) [reply about_us.md]: The creation of PP/SNR should be in there somewhere

File: about_us.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+Furthermore, we have few work in progress operators:
+
+- [Machine Deletion](/remediation/machine-deletion/machine-deletion) - Remediates nodes by deleting the associated OpenShift machine. Triggered by NHC.
+- [HA-SNO](https://github.com/medik8s/ha-sno) - High Availability Single Node Openshift.
```
Comment (mshitrit) [inline about_us.md]: Might want to mention fence agents as well - https://github.com/mshitrit/fence-agents-remediation

Comment (mshitrit) [inline about_us.md]: I think as a title it's a bit confusing, one can assume that somehow we miraculously achieved HA with only a single node.
Maybe instead: High Availability based on two Single Node Openshift ?

File: about_us.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+mechanisms to recover bad nodes.  Since then Ericsson has prototyped a
+[metal3](http://metal3.io/) based implementation, and we have implemented
+[Self Node Remediation](https://github.com/medik8s/self-node-remediation) for shared-nothing environments.
+
+In 2021, we created medik8s to make general purpose HA available
+to all kubernetes clusters, not just ones backed by an infrastructure API.
+
+
+## Community
+Join [our google group](https://groups.google.com/g/medik8s) to get more info, participate in dicussions and get notified
```
Comment (mshitrit) [inline about_us.md:67]: Maybe put first our public slack ? (I think it's probably used more)

**Comment by mshitrit** [comment]
I've put some comments.
Nothing major, as far as I'm concerned it's a good start and we can always build on it later, feel free to address any of the comments you'd like and merge.
/lgtm
/hold

File: about_us.md (hunk 4 above)
Comment (razo7) [reply about_us.md]: Yes, sounds better.

File: about_us.md (hunk 5 above)
Comment (razo7) [reply about_us.md:67]: It is indeed more used, but it is only for Red Hatters...

**Pull request merged**